### ISSUE TITLE
Carthage builds not working

### ISSUE THREAD

**Issue by ratkins** [pivotal/cedar#342]
I saw #328 and #330 which imply this should work now, but here's what I'm seeing:

<pre>
> cat Cartfile 
github "pivotal/cedar"

> carthage update
*** Fetching cedar
*** Checking out cedar at "v0.11.3"
Parse error: expected submodule commit SHA in ls-tree output: 

>
</pre>

It doesn't seem to be the obvious fix implied by https://github.com/Carthage/Carthage/issues/135 — I can't find anywhere that Cedar defines any submodules.

**Comment by tjarratt** [comment]
I've found that pinning to a particular SHA, rather than the version seems to work. I spent a few hours last weekend looking into that rather opaque Carthage error message and couldn't make heads or tails of it. It seems to be their way of saying "your directory structure doesn't conform to our expectations for what a project should look like".

For reference, here's what is currently working for me:

```bash
$ /tmp/cedar-carthage
○ cat Cartfile
github "pivotal/cedar" "9ac020e7ee1d82534122ca9290c88278ec69dc9d"
```

**Comment by tjarratt** [comment]
Interestingly enough, it seems when a SHA Is not provided, the version of Cedar that Carthage checks out doesn't include the top-level `Cedar-iOS-Framework` directory.

**Comment by akitchen** [comment]
Without a sha it may default to the last release. In our case that predates dylib support.




Sounds like a new release is needed, with a minor version bump perhaps.

**Comment by pivotal-brian-croom** [comment]
Yeah, Carthage looks for semantically versioned tags when deciding what to check out. A version bump with a new release tag should make this work out-of-the-box.

**Comment by tjarratt** [comment]
Thank you @akitchen and @pivotal-brian-croom for enlightening me.

Semantically, is this a minor version bump? The changes to Cedar's umbrella header that require files importing `Cedar.h` to be ObjC++ make me feel like this is really a major version bump.

**Comment by akitchen** [comment]
@tjarratt you're probably right.  v1.0.0 summons a certain amount of trepidation, but why not.

**Comment by tjarratt** [comment]
Hey @ratkins now that we've released [v0.12.0](https://github.com/pivotal/cedar/releases/tag/v0.12.0), including support for Carthage, do you feel comfortable closing this issue?

**Comment by ratkins** [comment]
Yup, no problem.
